examples/NonReg/SPI_loop/SPI_loop.ino

```diff
@@ -0,0 +1,81 @@
+/*
+  SPI_loopback
+
+  Test the loopback transfer on SPI serial link.
+  A test string is sent 9 times and compared to the Rx frame.
+  when the MISO pin is physically connected to the MOSI pin of the SPI bus
+  Default test the SPI_A link. To test the SPI_B transfer, just define SPI_B
+  to change the pin
+
+  This example code is in the public domain.
+
+*/
+#include "SPI.h"
+
+#if defined(SPI_B)
+/* when testing the SPI _B */
+ int MOSI_PIN = 22; // to be connected to the B_MISO
+ int SCK_PIN = 23;
+ int CS_PIN = 24;
+ int MISO_PIN = 25; // to be connected to the B_MOSI
+#else
+#define MOSI_PIN  PIN_SPI_MOSI
```

Comment:
By default test the default SPI instance and simply add a comment to explain to the user the way to test an other SPIx peripheral instance.